value/src/main/java/com/google/auto/value/processor/autovalue.vm

```diff
@@ -75,16 +75,14 @@ ${modifiers}class $subclass$formalTypes extends $origClass$actualTypes {
     ## the constructor is called from the extension code.
 
     #if ($identifiers)
-    if ($p == null) {
-      throw new NullPointerException("Null $p.name");
-    }
+        this.$p = `java.util.Objects`.requireNonNull($p, "Null $p");
```

Comment:
This change makes sense. However, the indentation convention in this file is that Velocity directives (like `#if`) and code lines are indented independently of each other. So this line should be indented the same as the `if` statement it replaces.

(It happens that the code indentation doesn't matter because a separate indentation pass will fix it up anyway, but I think following the conventions makes the code easier to follow.)

Reply:
Fixed indentation. Please let me know if I missed something.